checkov/secrets/runner.py

```diff
@@ -234,3 +235,12 @@ def search_for_suppression(
                     "suppress_comment": skip_search.group(3)[1:] if skip_search.group(3) else "No comment provided"
                 }
         return None
+
+    @staticmethod
+    def omit_secret_value_from_line(secret: str, line_text: str) -> str:
+        secret_index = line_text.index(secret)
+        secret_length = len(secret)
+        censored_line = line_text[:secret_index + 4] + \
```

Comment:
please use an f-string, because the current approach creates 3 strings in total of which are 2 just intermediate.
And what happens if the secret is 4 chars or less long?

Reply:
Changed to f-string, and exposing 1/4 of the secret instead of 4 chars